images/el8/Dockerfile.openshift

```diff
@@ -0,0 +1,45 @@
+#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_9_golang_1.21)
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.21 as builder
+
+WORKDIR /go/src/github.com/openshift/selinuxd
+
+ENV GOFLAGS="-mod=vendor" BUILD_FLAGS="-tags strictfipsruntime"
+
+COPY . .
+
+RUN yum -y update --allowerasing && \
+    yum -y install container-selinux git && yum clean all && \
+    yum -y clean all && rm -rf /var/cache/yum
+
+
+RUN SEMODULE_BACKEND=policycoreutils make
+
+FROM registry.redhat.io/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN INSTALL_PKGS="policycoreutils" && \
+    if [ ! -e /usr/bin/dnf ]; then ln -s /usr/bin/microdnf /usr/bin/dnf; fi && \
+        dnf install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+        dnf clean all && rm -rf /var/cache/* && \
+        # trigger to run pipeline for this image
+        rm -rf /var/cache/*
+
+LABEL \
+        io.k8s.display-name="selinux deamon for the Security Profiles Operator" \
+        io.k8s.description="An operand for the Security Profiles Operator" \
+        io.openshift.tags="selinux,security,deamon" \
+        com.redhat.delivery.appregistry="false" \
+        maintainer="Red Hat ISC <isc-team@redhat.com>" \
+        License="APL 2.0" \
+        name="openshift-selinuxd" \
+        com.redhat.component="openshift-selinuxd-container" \
+        io.openshift.maintainer.product="OpenShift Container Platform" \
+        io.openshift.maintainer.component="Security Profiles Operator" \
+        version=0.8.4
```

Comment:
Do we need to update the version here to a dev version of some kind? I though I hit issues with the version being the same as a previous release when building the OLM dependency graph.